.vscode/launch.json

```diff
@@ -1,403 +1,985 @@
-                    "--package=hotshot"
+                    "--features=full-ci,channel-async-std",
+                    "--package=hotshot-testing",
+                    "double_permanent_failure::MemoryCommChannel::MemoryStorage::VDEntryBlock::VDEntryState::double_permanent_failure"
                 ],
                 "filter": {
-                    "name": "multi-machine-libp2p",
-                    "kind": "example"
+                    "name": "hotshot-testing",
+                    "kind": "lib"
                 }
             },
-            "args": [],
-            "cwd": "${workspaceFolder}"
+            "program": "${cargo:program}",
+            "cwd": "${workspaceFolder}",
+            "name": "Debug double_permanent_failure::MemoryCommChannel::MemoryStorage::VDEntryBlock::VDEntryState::double_permanent_failure test"
         },
         {
             "type": "lldb",
             "request": "launch",
-            "name": "Debug example 'multi-machine'",
             "cargo": {
                 "args": [
-                    "build",
-                    "--example=multi-machine",
-                    "--package=hotshot"
+                    "test",
+                    "--no-run",
+                    "--features=full-ci,channel-async-std",
+                    "--package=hotshot-testing",
+                    "single_permanent_failure::MemoryCommChannel::MemoryStorage::VDEntryBlock::VDEntryState::single_permanent_failure"
                 ],
                 "filter": {
-                    "name": "multi-machine",
-                    "kind": "example"
+                    "name": "hotshot-testing",
+                    "kind": "lib"
                 }
             },
-            "args": [],
-            "cwd": "${workspaceFolder}"
+            "program": "${cargo:program}",
+            "cwd": "${workspaceFolder}",
+            "name": "Debug single_permanent_failure::MemoryCommChannel::MemoryStorage::VDEntryBlock::VDEntryState::single_permanent_failure test"
         },
         {
             "type": "lldb",
             "request": "launch",
-            "name": "Debug unit tests in example 'multi-machine'",
             "cargo": {
                 "args": [
                     "test",
                     "--no-run",
-                    "--example=multi-machine",
-                    "--package=hotshot"
+                    "--features=full-ci,channel-async-std",
+                    "--package=hotshot-testing",
+                    "libp2p_network"
                 ],
                 "filter": {
-                    "name": "multi-machine",
-                    "kind": "example"
+                    "name": "hotshot-testing",
+                    "kind": "lib"
+                }
+            },
+            "program": "${cargo:program}",
+            "cwd": "${workspaceFolder}",
+            "name": "Debug libp2p_network test"
+        },
+        {
+            "type": "lldb",
```

Comment:
That is a combinatoric explosion. A shame there's no alternative in this configuration.

Reply:
There might be a better way, but I'm not aware of it. Maybe we shouldn't even commit this and have the user generate the config file...